js/TopMenuModule.js

```diff
@@ -1,7 +1,7 @@
 const TopMenuModule = (() => {
 
-    const mainMenuItems = document.getElementById('main-menu').children;
-    let infoList = document.getElementById('editor-info');
+    const mainMenuItems = document.getElementById('main-menu')?.children ?? [];
```

Comment:
Same as the thing with the canvasView, if the elements are not there, we're only delaying the inevitable (same at line 4)